src/mochaVErrorReporter.ts

```diff
@@ -0,0 +1,47 @@
+import { Runner, utils as MochaUtils, reporters as MochaReporters } from "mocha";
+import { VError } from "verror";
+
+export class MochaVErrorReporter extends MochaReporters.Base
+{
+    private static _DEFAULT_INDENT: string = "    ";
```

Comment:
`private static readonly` for all

Reply:
Fixed. I also removed one of them since it was no longer needed